### ISSUE TITLE
Save mute option

### ISSUE THREAD

**Issue by leo60228** [v6cord/VVVVVV-CE#3]
@InfoTeddy I can't use Discord right now, but I couldn't figure out how to save accessibility options. You did that for the fake load screen, can you do the same thing for the music setting?

**Comment by InfoTeddy** [comment]
The loading of `unlock.vvv` is handled in `Game::loadstats()` and the saving of it is handled in `Game::savestats()`

Each tag in `Game::loadstats()` looks like
```c++
if (pKey == "thingVar")
{
    thingVar = atoi(pText);
}
```
and each tag in `Game::savestats()` looks like
```c++
    msg = new TiXmlElement("thingVar");
    msg->LinkEndChild(new TiXmlText(tu.String(thingVar).c_str()));
    dataNode->LinkEndChild(msg);
```
It's all easily copy-pastable code.

**Comment by InfoTeddy** [comment]
Closed as of ffb28a6f352faefd7c5c2b36a1524cc3e6d8ff36.
